server/model/User.js

```diff
@@ -1,6 +1,10 @@
+const { v1: uuid } = require('uuid');
+
 class User {
```

Comment:
Are you missing the number of losses? If hands only includes wins, there might be a better variable name.

Reply:
My bad, forgot to  pull master before I worked on this. Fixed